tools/ocean/divandfull.xml

```diff
@@ -58,7 +58,7 @@
         <param name="selmax" type="integer" min="0" max="100" optional="true" value="0" label="Maximum of the salinity"/>
     </inputs>
     <outputs>
-        <data name="output_netcdf" label="DIVAnd netcdf output" from_work_dir="outputs_netcdf/*.nc" format="netcdf"/>
+        <data name="output_netcdf" label="DIVAnd_output.nc" from_work_dir="outputs_netcdf/*.nc" format="netcdf"/>
```

Comment:
the previous label was better :)

Why are you bumping this tool?

Reply:
Weird.. should I close this PR and change my branch and open a new PR ?

Reply:
okay I rolled back those weird commits ! no problem there anymore